basket/settings.py

```diff
@@ -396,54 +392,48 @@ def before_send(event, hint):
 
 # language codes that we support and send through to backend
 # regardless of their existence in the DB
-EXTRA_SUPPORTED_LANGS = config("EXTRA_SUPPORTED_LANGS", "", cast=Csv())
+EXTRA_SUPPORTED_LANGS = config(
+    "EXTRA_SUPPORTED_LANGS",
+    parser=ListOf(str, allow_empty=False),
+    default="",
+)
 
 if UNITTEST:
     TESTING_EMAIL_DOMAINS = []
 else:
     TESTING_EMAIL_DOMAINS = config(
         "TESTING_EMAIL_DOMAINS",
-        "restmail.net,restmail.lcip.org,example.com",
-        cast=Csv(),
+        parser=ListOf(str, allow_empty=False),
+        default="restmail.net,restmail.lcip.org,example.com",
```

Comment:
What's `restmail.lcip.org`?

Reply:
It was a restmail instance, and it looks like Mozilla still owns `lcip.org`, but nothing is there anymore. May be outdated.

Reply:
Probably best to remove it if nothing lives there.